phpcs.xml.dist

```diff
@@ -30,7 +30,7 @@
     https://github.com/PHPCompatibility/PHPCompatibility
     -->
     <config name="testVersion" value="7.0-"/>
-    <config name="minimum_supported_wp_version" value="4.9"/>
+    <config name="minimum_supported_wp_version" value="5.6"/>
```

Comment:
how would customers running PHP < 5.6 be affected by this change?

Reply:
No effect on customers as this is just for our internal tests. Our plugin supports WP 5.4.4 and higher presently, so we don't have any customers running 4.X actively. As 4.X has been EOL for several years and we don't have any users on it, we can safely bump our testing to only target modern versions.